metashare/api/views.py

```diff
@@ -35,6 +37,25 @@ def get_object(self):
         return self.get_queryset().get()
 
 
+class HookView(APIView):
+    authentication_classes = (GitHubHookAuthentication,)
+
+    def post(self, request):
+        # To support the various formats that GitHub can post to this endpoint:
+        # TODO: we can route this based on the X-GitHub-Event header.
```

Comment:
@wlonk Is this waiting on anything?

Reply:
I think I'll flesh that out in a branch where I actually handle PR events, too. As it stands, it's a little hard to clearly handle when we're only supporting one event type.